entity/BaseEntity.php

```diff
@@ -138,10 +143,12 @@ public function __call(string $method, array $parameters)
             array_unshift($parameters, $key);
 
             //call getAttribute
-            if ($methodParts[0] == 'get') return $this->getProperty(...$parameters);
+            if ($methodParts[0] == 'get'){
+                return $this->getProperty(...$parameters);
+            };
 
             //call setAttribute
-            if ($methodParts[0] == 'set') return $this->setProperty(...$parameters);
+            return $this->setProperty(...$parameters);
```

Comment:
This change makes the ‘set’ operation the default. Is it safe in all cases?

Reply:
It is safe because the condition at line 136 suggests that $methodParts[0] should only contain "get" or "set".